common/src/main/java/com/gregtechceu/gtceu/integration/rei/oreprocessing/GTOreProcessingDisplay.java

```diff
@@ -0,0 +1,45 @@
+package com.gregtechceu.gtceu.integration.rei.oreprocessing;
+
+import com.gregtechceu.gtceu.api.data.chemical.ChemicalHelper;
+import com.gregtechceu.gtceu.api.data.chemical.material.Material;
+import com.gregtechceu.gtceu.integration.GTOreProcessingWidget;
+import com.lowdragmc.lowdraglib.gui.widget.WidgetGroup;
+import com.lowdragmc.lowdraglib.rei.ModularDisplay;
+import me.shedaniel.rei.api.common.entry.EntryIngredient;
+import me.shedaniel.rei.api.common.entry.EntryStack;
+import me.shedaniel.rei.api.common.entry.type.VanillaEntryTypes;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.gregtechceu.gtceu.api.data.tag.TagPrefix.*;
+
+public class GTOreProcessingDisplay extends ModularDisplay<WidgetGroup> {
+
+    private final Material material;
+
+    public GTOreProcessingDisplay(Material material) {
+        super(() -> new GTOreProcessingWidget(material), GTOreProcessingDisplayCategory.CATEGORY);
+        this.material = material;
+    }
+
+    @Override
+    public List<EntryIngredient> getInputEntries() {
+        List<EntryIngredient> ingredients = new ArrayList<>();
+        ingredients.add(EntryIngredient.of(EntryStack.of(VanillaEntryTypes.ITEM, ChemicalHelper.get(ore, material))));
+        ingredients.add(EntryIngredient.of(EntryStack.of(VanillaEntryTypes.ITEM, ChemicalHelper.get(rawOre, material))));
```

Comment:
this could be a tag? like `EntryIngredients.ofItemTag(ChemicalHelper.getTag(ore, material))`